src/utils/GraphhopperUtils.js

```diff
@@ -143,14 +143,13 @@ function busRoutesAreEqual(busRouteA: Object, busRouteB: Object): boolean {
 function getValidBusRoutes(busRoutes: Array<Object>, busRoutesToCheck: Array<Object>): Array<Object> {
   return busRoutesToCheck.filter((busRouteToCheck) => {
     const isDuplicateRoute = busRoutes.find(busRoute => busRoutesAreEqual(busRouteToCheck, busRoute)) !== undefined;
-    return !isDuplicateRoute;
+    return !isDuplicateRoute && busRouteToCheck.constructor.transfers !== -1;
   });
```

Comment:
this isn't part of the PR but why would there be duplicate routes?

Reply:
since we do buffered requests, a request for 2 minutes after might return the same route as making a request for now